UnityProject/Assets/Scripts/NPC/AI/Friendly/ChickenAI.cs

```diff
@@ -47,14 +40,14 @@ protected override void OnSpawnMob()
 
 		protected override void DoRandomAction()
 		{
-			// Check if we're a happy chiken/chick or we alreday laid all possible egs
-			if (mood.Level < happyChicken || currentLaidEggs == maxEggsAmount)
+			// Check if we alreday laid all possible egs
+			if (currentLaidEggs == maxEggsAmount)
```

Comment:
```suggestion
			if (currentLaidEggs >= maxEggsAmount)
```

Just a small thing, but just a safety in consideration for VVing.